libraries/lib-wave-track/CMakeLists.txt

```diff
@@ -35,8 +35,6 @@ set( SOURCES
    WaveTrackSink.h
    WaveTrackUtilities.cpp
    WaveTrackUtilities.h
-   WideClip.cpp
```

Comment:
Great to see these gone!
I was expecting this commit to justify the previous one, a67b7aded083eea4a3fbdfb455e95801e49a60f6, but does it, actually?

Reply:
Yes, it simplified the job, avoiding even more const-casting in WaveTrack.cpp